core/src/main/scala/weaponregex/mutator/BuiltinMutators.scala

```diff
@@ -39,17 +39,26 @@ object BuiltinMutators {
   /** Map from mutation level number to token mutators in that level
     */
   @JSExport
-  lazy val levels: Map[Int, Seq[TokenMutator]] =
+  lazy val asMap: Map[Int, Seq[TokenMutator]] =
```

Comment:
Is this a breaking API change?

Reply:
It is, I'll update the PR message.
Also, I just realized that those are not really for JS users since they are not of native JS types, so I created `BuiltinMutatorsJS` for that.

Reply:
For breaking changes, I think it's better to be careful with them. It can be pretty difficult to migrate to a newer version if certain functions have suddenly disappeared. For any renames, I'd suggest either keeping the old function and making it `@depcrecated` with a message of what the replacement is, or keeping it altogether

Reply:
Understood 👍 I'll add the old functions back with `@deprecated` with a message says that they will be removed in later version, maybe `v1.0.0`.

For StrykerJS, I doubt that anything in `BuiltinMutators` was used since they are not in native JS types 😅 which I created `BuiltinMutatorsJS` to do.